hibernate-core/src/main/java/org/hibernate/dialect/MariaDBDialect.java

```diff
@@ -51,9 +50,12 @@ public MariaDBDialect(DialectResolutionInfo info) {
 
 	@Override
 	public DatabaseVersion getMySQLVersion() {
-		return getVersion().isBefore( 5, 3 )
-				? VERSION5
-				: VERSION57;
+		return super.getVersion();
```

Comment:
This should always return `VERSION57`

Reply:
I wasn't 100% sure what to do here since the MariaDB dialect extends the MySQL one, and currently that one is on minimum 5.7, but what if in the future we update MySQl to a newer version? I assumed that the MariaDB dialect would simply follow suit for `getMySQLVersion()`, hence the `super.getVersion()`